docs/conf.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+
+sys.path = ['.', '..'] + sys.path
+
+project = 'AutoGluon-Cloud'
+release = '0.1.1'
+copyright = '2023, All authors. Licensed under Apache 2.0.'
+author = 'AutoGluon contributors'
+
+extensions = [
+    'myst_nb',                       # myst-nb.readthedocs.io
+    'sphinx_copybutton',             # sphinx-copybutton.readthedocs.io
+    'sphinx_design',                 # github.com/executablebooks/sphinx-design
+    'sphinx_inline_tabs',            # sphinx-inline-tabs.readthedocs.io
+    'sphinx_togglebutton',           # sphinx-togglebutton.readthedocs.io
+    'sphinx.ext.autodoc',            # www.sphinx-doc.org/en/master/usage/extensions/autodoc.html
+    'sphinx.ext.autosummary',        # www.sphinx-doc.org/en/master/usage/extensions/autosummary.html
+    'sphinx.ext.napoleon',           # www.sphinx-doc.org/en/master/usage/extensions/napoleon.html
+    'sphinx.ext.viewcode',           # www.sphinx-doc.org/en/master/usage/extensions/viewcode.html
+    'sphinxcontrib.googleanalytics', # github.com/sphinx-contrib/googleanalytics
+    ]
+
+# See https://myst-parser.readthedocs.io/en/latest/syntax/optional.html
+myst_enable_extensions = ['colon_fence', 'deflist', 'dollarmath', 'html_image', 'substitution']
+
+autosummary_generate = True
+numpydoc_show_class_members = False
+
+googleanalytics_id = "UA-96378503-20"
+
+nb_execution_mode = 'force'
```

Comment:
You can set this to `off` while you don't have any notebooks that need to be executed.

Reply:
I think we can leave it as is cause it's possible that we will want to execute some content in the future, and it would be confusing for someone unfamiliar with this to know how to enable execution